src/main/java/org/javarosa/core/model/condition/Recalculate.java

```diff
@@ -101,16 +110,6 @@ public boolean equals(Object o) {
         }
     }
 
-    //droos 1/29/10: we need to come up with a consistent rule for whether the resulting data is determined
```

Comment:
Can we keep this in `wrapData` for now? I don't think those rules are documented anywhere else.

Reply:
Sure!